### DISCUSSION TITLE
Is there a way to concurrently use udm-iptv (solcon) with openvpn client (surfshark)?

### DISCUSSION THREAD

**Discussion by rberretty** [fabianishere/udm-iptv#287] [Q&A]
I seem to have some routing issues with my vpn connection when I use both udm-iptv and surfshark openvpn client connection on the udm pro

Destination     Gateway         Genmask         Flags Metric Ref    Use Iface
10.0.0.0        10.71.0.1       255.0.0.0       UG    274    0        0 iptv
10.8.8.0        0.0.0.0         255.255.255.0   U     0      0        0 tunovpnc1

openvpn stops connecting when I activate udm-iptv. I suspect the first entry in the routing table, but could not find a way to change the mask. Is there a way to fix this?

Thanks!

Robert-Paul

=== Configuration ===
WAN Interface: eth8
WAN VLAN: 4 (dev iptv)
WAN DHCP: true (options "-O staticroutes -V IPTV_RG")
WAN Ranges: 10.71.0.0/16
LAN Interfaces: br3
IGMP Proxy quickleave disabled: false
IGMP Proxy debug: false
=== IP Link and Route ===
74: iptv@eth8: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
    inet 10.71.10.195/20 scope global iptv
       valid_lft forever preferred_lft forever
10.0.0.0/8 via 10.71.0.1 metric 274 
10.71.0.0/20 proto kernel scope link src 10.71.10.195 
=== Service Logs ===
Nov 25 21:13:51 DeKadeUDM-Pro udm-iptvd[2382712]: Obtaining IP address for VLAN interface
Nov 25 21:13:51 DeKadeUDM-Pro udm-iptvd[2382721]: udhcpc: started, v1.30.1
Nov 25 21:13:51 DeKadeUDM-Pro udm-iptvd[2382721]: udhcpc: sending discover
Nov 25 21:13:52 DeKadeUDM-Pro udm-iptvd[2382721]: udhcpc: sending select for 10.71.10.195
Nov 25 21:13:52 DeKadeUDM-Pro udm-iptvd[2382721]: udhcpc: lease of 10.71.10.195 obtained, lease time 86400
Nov 25 21:13:52 DeKadeUDM-Pro udm-iptvd[2382712]: NATing IPTV network ranges (if necessary)
Nov 25 21:13:52 DeKadeUDM-Pro udm-iptvd[2382712]: Creating static routes (if necessary)
Nov 25 21:13:52 DeKadeUDM-Pro udm-iptvd[2382712]: Setting up IGMP Proxy
Nov 25 21:13:52 DeKadeUDM-Pro udm-iptvd[2382712]: Using improxy...
Nov 25 21:13:53 DeKadeUDM-Pro udm-iptvd[2382712]: Starting IGMP Proxy

**Comment by fabianishere** [comment]
The routes from your IPTV provider and VPN overlap. Would it be possible to change the range of your VPN server? The 10.0.0.0/8 is sent by your ISP, so I suspect you need that route for IPTV to work.
